docs/UserGuide.md

```diff
@@ -249,27 +233,21 @@ view NAME
   e.g. if an applicant's full name is `Steve Jobs`, and the command provided is `view Steve` then LinkedOUT will not
   display `Steve Jobs`' details. <br>
```

Comment:
Is it possible to help add `Steve Jobs` '**s** here? thanks

Reply:
Sure, will add